Docs/AcpiSamples/SSDT-SBUS-MCHC.dsl

```diff
@@ -8,9 +8,12 @@ DefinitionBlock ("", "SSDT", 2, "ACDT", "MCHCSBUS", 0x00000000)
 
     Scope (_SB.PCI0)
     {
+    If (_OSI ("Darwin"))
```

Comment:
This is way too fugly. Try to avoid conditional declarations as they may be harder to handle for the ACPI implementation. Better use STA method here and elsewhere.

Reply:
on mchc and imei better the method _sta? as you like ok 💪🙈

Reply:
Well, the issue is rather conceptual. The device exists, it is just that you do not want to use it e.g. in Windows.